src/features/metadata-instance-editor/Instances.js

```diff
@@ -14,27 +14,50 @@ type Props = {
         cascadingPolicy?: MetadataCascadingPolicyData,
         rawData: Object,
     ) => void,
+    templateFilters?: MetadataTemplateFilters,
 };
 
-const Instances = ({ isCascadingPolicyApplicable = false, editors = [], onModification, onRemove, onSave }: Props) =>
+const Instances = ({
+    isCascadingPolicyApplicable = false,
+    editors = [],
+    onModification,
+    onRemove,
+    onSave,
+    templateFilters,
+}: Props) =>
     editors.map<React.Element<typeof Instance>>(
-        ({ isDirty = false, instance, hasError = false, template }: MetadataEditor) => (
-            <Instance
-                canEdit={instance.canEdit}
-                cascadePolicy={instance.cascadePolicy}
-                data={instance.data}
-                hasError={hasError}
-                id={instance.id}
-                isCascadingPolicyApplicable={isCascadingPolicyApplicable}
-                isDirty={isDirty}
-                isOpen={editors.length === 1}
-                key={`${instance.id}-${template.templateKey}`}
-                onModification={onModification}
-                onSave={onSave}
-                onRemove={onRemove}
-                template={template}
-            />
-        ),
+        ({ isDirty = false, instance, hasError = false, template }: MetadataEditor) => {
+            const { templateKey } = template;
+            let isOpen = false;
+            let includedFieldIds;
+            // Open the included template by default, and only display the included fields in the template
+            if (templateFilters) {
+                isOpen = templateKey === templateFilters.includedTemplateKey;
```

Comment:
It looks like `includedTemplateKey` only controls whether the template is open by default. Should it be a separate prop rather than part of `templateFilters`?